paper/paper.md

```diff
@@ -43,60 +43,85 @@ bibliography: paper.bib
+Our primary motivation to develop new software was the combination of the following needs:
 
-# Statement of Need
+* The level of detail sufficient to allow for an investigation of transient effects of thermal storage.
+* The simulation time for a single system short enough to allow for over 100 thousand simulations to be performed on a personal computer within a reasonable amount of time, as this is how many we needed to be performed in a single state-level analysis run.
+* Simplicity of integration within the larger life-cycle cost framework as presented in @Coughlin:2020 and @Grahovac:2020.
+
+Modelica buildings library by @Wetter:2014 exceeds the level of detail but proves too detailed and thus somewhat slow for our particular application. SAM tool [@Blair:2014] has a fitting level of detail, provides most of the system models that we needed but for our purposes proves not flexible enough in terms of modifying the system configuration, automating the size scaling, and embedding it into our custom life-cycle cost framework.
 
-The project that prompted the development of this software, described in @Coughlin:2020, that is at the moment of this writing in press, was enquiring whether there are any economic benefits in grouping households to be served by a single solar water heating installation, in comparison to single household solar water heating installations on the state of California level.
+Namely, in order to capture a sufficient level of detail of the California demographics, such as variability in climate zones, household types, and household occupancy, we wanted to be able to simulate a few alternative water heating systems in each of the California sample households. Secondly, to get a more realistic picture of the effect of thermal storage and distribution system losses, we opted to perform a simulation with relatively short time-steps of 1 hour for a duration of one representative year. We were not able to identify an open source tool that is capable of firstly satisfying the simulation speed requirement combined with the necessary level of detail for our analysis and secondly providing the flexibility for us to customize various integral parts of the analysis such as automate the component and system size scaling, specify hot water load profiles and solar radiation for each household or group of households in the sample.
 
-Our primary motivation to develop a new software was the combination of the following factors:
+To satisfy our research need we thus opted to develop lightweight simulation models for all involved systems that would allow for around 120 thousand simulation runs together with the component sizing and life-cycle cost analysis to be performed on a computer with a 12-core processor in about 8 hours. The users can expect a single solar water heater simulation model to run for about 0.16 seconds, providing an almost instantaneous experience for a user only seeking to design and investigate a single system.
```

Comment:
120 thousand >> 120,000

Also, the detail you put in here about running in 0.16 seconds is highly computer-specific.  You might make it more general by saying that, "The users can expect a single solar water heater simulation model to run in less than one second (the developers were experiencing run times on the order of 0.2 seconds), providing an almost instantaneous..."